skeleton/protractor/protractor.conf.js

```diff
@@ -56,7 +56,8 @@ const config  = {
         '--disable-renderer-backgrounding',
         '--disable-device-discovery-notifications',
         '--no-sandbox',
-        '--disable-dev-shm-usage'
+        '--disable-dev-shm-usage',
+        '--window-size=1024,768'
```

Comment:
@shahabganji this fix did the job. The reason for `element not interactable` is that the protractor browser window size is too small.

Nobody knows why the default screen size is so small inside docker(or maybe vm). Explicit window size does the trick.

Reply:
When element is off screen, protractor says it's `not interactable`.